src/pages/api/users/addcard.js

```diff
@@ -0,0 +1,28 @@
+import dbConnect from "@/utils/dbConnect";
+import { modifyUserPaymentMethod } from "@/utils/dbFunctions"
+
+export default async function handler(req, res)
+{
+    const { method } = req;
+
+    await dbConnect();
+
+    switch (method)
+    {
+        case "PATCH":
```

Comment:
I personally really don't like `switch` cases and I never use them. I don't ever feel the need for them, and I think they're unnecessarily verbose, too "wide" (adds extra indentations) and also error-prone. A common mistake is to forget a `break`, which is completely valid (it's called "fall-through" and considered a feature) but very rarely what you want.